CMakeLists.txt

```diff
@@ -113,6 +113,17 @@ set(codegen_scripts
 set(nidriver_service_srcs "")
 set(nidriver_client_srcs "")
 
+# We'll codegen all drivers, but some are not included in the server build.
```

Comment:
There's something wrong with the way I have this change set up in github. This block should not be showing up in my change, as it's already in grpc-device.

Reply:
Fixed the merge conflicts.